src/routes/safe/components/Settings/ManageOwners/RemoveOwnerModal/index.jsx

```diff
@@ -0,0 +1,179 @@
+// @flow
+import React, { useState, useEffect } from 'react'
+import { List } from 'immutable'
+import { withStyles } from '@material-ui/core/styles'
+import { SharedSnackbarConsumer } from '~/components/SharedSnackBar'
+import Modal from '~/components/Modal'
+import { type Owner } from '~/routes/safe/store/models/owner'
+import { getGnosisSafeInstanceAt } from '~/logic/contracts/safeContracts'
+import CheckOwner from './screens/CheckOwner'
+import ThresholdForm from './screens/ThresholdForm'
+import ReviewRemoveOwner from './screens/Review'
+
+const styles = () => ({
+  biggerModalWindow: {
+    width: '775px',
+    minHeight: '500px',
+    position: 'static',
+  },
+})
+
+type Props = {
+  onClose: () => void,
+  classes: Object,
+  isOpen: boolean,
+  safeAddress: string,
+  safeName: string,
+  ownerAddress: string,
+  ownerName: string,
+  owners: List<Owner>,
+  threshold: number,
+  network: string,
+  createTransaction: Function,
+  removeSafeOwner: Function,
+}
+type ActiveScreen = 'checkOwner' | 'selectThreshold' | 'reviewRemoveOwner'
+
+const SENTINEL_ADDRESS = '0x0000000000000000000000000000000000000001'
```

Comment:
this address is also duplicated in `src/routes/safe/components/Settings/ManageOwners/ReplaceOwnerModal/index.jsx`
Let's put it  in as const in `src/logic/contracts/safeContracts.js` maybe?